src/Helpers/Click/ClickedArtworkGroup.ts

```diff
@@ -9,8 +9,8 @@ import {
 export interface ClickedArtworkGroupArgs {
   contextModule: ContextModule
   contextPageOwnerType: PageOwnerType
-  contextPageOwnerId: string
-  contextPageOwnerSlug: string
+  contextPageOwnerId?: string
```

Comment:
Which page is this being used on that has no id/slug?

Reply:
artsy.net/consign